routes/index.js

```diff
@@ -563,7 +572,7 @@ router.put('/discover', function(req, res, next) {
     //apply 'show songs in' filter
     var loadMore = function(songs2d, done) {
         if (langShown !== 'all') {
-            songs2d = songs2d.filter((s) => s.lang === langShown)
+            songs2d = songs2d.filter((s) => s.lang == langShown)
```

Comment:
why did you drop the triple === ? 
We generally want the comparison to be strict. You can compare the label if needed, but comparing id would be better.